operations/serviceorders.md

```diff
@@ -0,0 +1,75 @@
+# Service orders
+
```

Comment:
I think it would be helpful to insert a note here. Please read the wording and confirm that my understanding is correct.

> **Service Orders:** A Service Order is an Order made against a Service. There may be different types of Service Order, but the most common is a Reservation, which is an Order made against a Bookable Service. Note that a Reservation ID can be used as a Service Order ID in Service Order operations such as [Get all service order notes](#get-all-service-order-notes).

Reply:
Left blank on purpose, you don't have those new endpoint available yet